src/applications/claims-status/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Might be helpful here to add a comment about why we are changing the component library to this version, incase someone tries to change is, especially now that we have two different teams working in this area!